xml/schema/CRM/Aip/AipErrorLog.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+<table>
+  <base>CRM/Aip</base>
+  <class>AipErrorLog</class>
+  <name>civicrm_aip_error_log</name>
+  <comment>AIP Process</comment>
+  <log>false</log>
+
+  <field>
+    <name>id</name>
+    <type>int unsigned</type>
+    <uniqueName>aip_error_log_id</uniqueName>
```

Comment:
The documentation says `uniqueName` should only be used for core entities... https://docs.civicrm.org/dev/en/latest/framework/database/schema-definition/

Reply:
removed them with fa7f36730b2f5d8a718cc009c5e62ba98c9b0781